requirements/detection_test.txt

```diff
@@ -0,0 +1 @@
+pycocotools
```

Comment:
i guess this should technically be in a `requirements/detection.txt` file because the metric cannot be used without it when `iou_type='segm'`

Reply:
so we lost independence on `pycocotools`? that is not good, it shall be at least optional...

Reply:
```suggestion
```